security/cert_manager_operator/cert-manager-customizing-api-fields.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="cert-manager-customizing-api-fields"]
+= Customizing cert-manager Operator API fields
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-customizing-api-fields
+
+toc::[]
+
+You can customize the {cert-manager-operator} API fields by overriding environment variables and arguments.
+
+[WARNING]
+====
+You can override only limited supported arguments in the {cert-manager-operator}. You cannot override the default arguments.
```

Comment:
``
`You cannot override the default arguments` is precisely incorrect. Users can (and sometimes want to) do it per @swghosh https://github.com/openshift/openshift-docs/pull/59726#discussion_r1208897243 .
```suggestion
You can override only limited supported arguments in the {cert-manager-operator}. You cannot override the default arguments.
```